src/components/Offers/Edit/EditOfferForm.js

```diff
@@ -41,9 +43,18 @@ const parseOfferForm = ({
     vacancies: vacancies || "",
     description,
     descriptionText: parseDescription(description),
+    applyURL: /^mailto:/.test(applyURL) ? applyURL.substring(7) : applyURL,
     ...offer,
 });
 
+const parseApplyURL = (applyURL) => {
+    if (!applyURL)
+        return null;
+    if (MailRegex.test(applyURL) && /^(?!mailto:)/.test(applyURL))
```

Comment:
Just a suggestion, maybe this function would be more readable? It should give the same result without regex: https://www.w3schools.com/jsref/jsref_startswith.asp

What do you think?

Reply:
Yes, it would make the code way more clean.